Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -1978,12 +1978,12 @@ def gen2018HiMix(fragment,howMuch):
                                    '--datatier' : 'MINIAODSIM',
                                    '--eventcontent':'MINIAOD',},stepMiniAODMC])
 
+stepNanoAODDefaults = { '-s': 'NANO,DQM:@nanoAODDQM', '-n': 1000 }
+stepNanoAODData = merge([{ '--data':'', '--eventcontent' : 'NANOAOD,DQM' ,'--datatier': 'NANOAOD,DQMIO'    }, stepNanoAODDefaults ])
```

Comment:
hi @arizzi - my old and perhaps outdated/wrong understanding is that DQM can only be run in one step of a workflow for some technical reason. [dating from the miniAOD dqm saga]

Reply:
I'm not sure I understand the point... you mean "running in the same cmsRun" ? Perhaps for miniaod it was related to validation of the lossy compression?